src/partials/Channel.svelte

```diff
@@ -64,7 +64,11 @@
   let groupedMessages = []
 
   const pubkeysWithoutInbox = derived(inboxRelaySelectionsByPubkey, $inboxRelayPoliciesByPubkey =>
-    pubkeys.filter(pubkey => !$inboxRelayPoliciesByPubkey.has(pubkey)),
+    pubkeys.filter(
+      pubkey =>
+        !$inboxRelayPoliciesByPubkey.has(pubkey) ||
+        !$inboxRelayPoliciesByPubkey.get(pubkey).publicTags.length,
+    ),
```

Comment:
You can use `getListTags($inboxRelayPoliciesByPubkey.get(pubkey)).length === 0` without having to check `has` since `getListTags` accepts undefined.